docs/settings.rst

```diff
@@ -774,6 +774,16 @@ be filtered out.
 
 For example: ``[(logging.WARN, 'TAG_SAVE_AS is set to False')]``
 
+It's possible to filter out messages by a template. Check out source code to obtain a template.
+
+For example: ``[(logging.WARN, 'Empty alt attribute for image %s in %s')]``
+
+**Warning:** Silence messages by templates is dangerous feature. There are odds
+to unintantional filter out messages of different types with the same template
+(includes future messages from newer Pelican versions). Proceed with caution.
```

Comment:
When I said "suppress future messages", I didn't mean any *new* warnings with same template (although quite unlikely, that is still a possibility). But I mean the same warning for future runs. Let's say you did the above and silenced `alt` warnings. You'll *never* see that warning again. It might be OK for now, since you made sure every warned image is supposed to have an empty alt. But that warning might be relevant in the future for new articles/pages.